packages/fxa-content-server/app/scripts/views/confirm_signup_code.js

```diff
@@ -72,6 +72,7 @@ class ConfirmSignupCodeView extends FormView {
     const newsletters = account.get('newsletters');
     const options = {
       service: this.relier.get('service') || null,
+      scopes: this.relier.get('permissions') || null,
```

Comment:
Client stores the scopes in an array called `permissions` and single string `scope` that stores a concatenation of all scopes. Using the array was a little easier to handle on the auth-server side of things.

Reply:
I think `permissions` may have some amount of post-processing done to it, e.g. removing scopes that aren't allowed or something like that? Anyway, it seems fine for this purpose.